src/modules/endpoint/models/CategoriesMapModel.js

```diff
@@ -48,7 +36,7 @@ class CategoriesMapModel {
    */
   getChildren (category) {
     return this.toArray()
-      .filter(_category => _category.parentUrl === category.url)
+      .filter(_category => _category.parentPath === category.path && _category !== category)
```

Comment:
Why is it required to check `_category !== category`? Because the `parentPath` of the root category is  also `/`?

Reply:
Yes, exactly. Otherwise a page would be displayed as it's own child.